app/2.0/docs/devguide/style-shadow-dom.md

```diff
@@ -274,54 +274,49 @@ Styling with the `:host` selector is one of two instances where rules inside a s
 
 ### Style slotted content (distributed children)
 
-You can create slots in an element's template that are populated at runtime using this syntax:
+You can create **slots** in an element's template that are populated at runtime. For more information on slots, see the documentation on [/2.0/docs/devguide/shadow-dom#shadow-dom-and-composition](shadow DOM and composition). 
 
-[See it on Plunker](http://plnkr.co/edit/MRN9blKg6A3w8G0RkyJD?p=preview)
+The basic syntax for incorporating slotted content looks like this:
+
+[See it on Plunker](http://plnkr.co/edit/bNvOvQqCEmC4DaoeNtwZ?p=preview)
 
 `x-foo.html` { .caption}
 ```html
 <dom-module id="x-foo">
   <template>
-    <style>
-      :host(.warning) p {
-        color: red;
-      }
-    </style>
-    <p>Make this text red if x-foo has class "warning", and black otherwise.</p>
+    <h1>
+      <slot name="title"></slot>
+    </h1>
   </template>
+  ...
 </dom-module>
-<script>
-  class XFoo extends Polymer.Element {
-    static get is() {
-      return "x-foo";
-    }
-  }
-  customElements.define(XFoo.is, XFoo);
-</script>
 ```
 
 `index.html` { .caption}
 ```html
 <link rel="import" href="x-foo.html">
 
-<x-foo class="warning"></x-foo>
-<x-foo></x-foo>
+<x-foo>
+  <span slot="title">I'm a heading!</span>
+</x-foo>
 ```
 
-To style slotted content, use the `::slotted()` syntax.
+To style slotted content, use the `::slotted()` syntax. 
+
+**Note:** To work within the Shady CSS scoping shim limitations, and to ensure consistent cross-browser behavior, add a selector to the left of the `::slotted(.classname)` notation (for example, `p ::slotted(.classname)`.
 
 `::slotted(*)` selects all slotted content:
 
-[See it on Plunker](http://plnkr.co/edit/cGrbntuantGtqj7Poqft?p=preview)
+[See it on Plunker](http://plnkr.co/edit/uA4bHmAFWgFjOSghEFeT?p=preview)
 
 `x-foo.html` { .caption}
 ```html
 <dom-module id="x-foo">
   <template>
     <style>
-      ::slotted(*) {
+      p, h1 ::slotted(*) {
```

Comment:
Sorry I missed this before: This rule may not be doing what you expect it to do. 
The comma is an "or" operator, but it operates on an entire selector. In other words: `p, h1 ::slotted(*) { ... } ` is equivalent to:

```
p { ... } 
h1 ::slotted(*) { ...}
```

And not as you might think:

```
p ::slotted(*) { ... } 
h1 ::slotted(*) { ...}
```